tests/system/test_observability_options.py

```diff
@@ -241,30 +272,70 @@ def select_in_txn(txn):
 
             got_events.append((event.name, evt_attributes))
 
+    return got_statuses, got_events
+
+
+@pytest.mark.skipif(
+    not _helpers.USE_EMULATOR,
+    reason="Emulator needed to run this test",
+)
+@pytest.mark.skipif(
+    not HAS_OTEL_INSTALLED,
+    reason="Tracing requires OpenTelemetry",
+)
+def test_database_partitioned():
```

Comment:
Looks like this is a negative case.
Can we test for one successful case for partitioned dml?